aspnetcore/blazor/includes/security/troubleshoot.md

```diff
@@ -3,10 +3,34 @@ no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Bla
 ---
 ## Troubleshoot
 
-::: moniker range=">= aspnetcore-5.0"
-
 ### Common errors
 
+* Misconfiguration of the app or Identity Provider (IP) service
+
+  The most common errors are caused by incorrect configuration. The following are a few examples:
+  
+  * Depending on the requirements of the scenario, a missing or incorrect Authority, Instance, Tenant ID, Tenant domain, Client ID, or Redirect URI prevents an app from authenticating clients.
+  * An incorrect access token scope prevents clients from accessing server web API endpoints.
+  * Incorrect or missing server API permissions prevent clients from accessing server web API endpoints.
+  
+  Configuration sections of this article's guidance show examples of the correct configuration. Carefully check each section of the article looking for app and IP misconfiguration.
+  
+  If the configuration appears correct:
+  
+  * Analyze application logs.
+  * Examine the network traffic between the client app and the IP or server app with the browser's developer tools. Often, an exact error message or a message with a clue to what's causing the problem is returned to the client by the IP or server app after making a request.
```

Comment:
Thinking about a new developers (not just to the .NET ecosystem), they may not be much familiar with browser dev tools. It's worth to link to different browser docs here on how to show network traffic: 
- Edge: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/network/
- Chrome: https://developers.google.com/web/tools/chrome-devtools/network/4
- Firefox: https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor